vendor/vendor-test/src/main/kotlin/com/malinskiy/marathon/test/StubDevice.kt

```diff
@@ -43,6 +45,10 @@ class StubDevice(
     ) {
         delay(testTimeMillis)
 
+        if (crashWithTestBatchException) {
+            throw TestBatchExecutionException("why not?")
```

Comment:
The exception message is useless. Please add more information

Reply:
fixed